code/game/objects/items/weapons/power_cells.dm

```diff
@@ -112,6 +112,27 @@
 	w_type = RECYK_BIOLOGICAL
 	minor_fault = 1
 
+/obj/item/weapon/cell/crepe
+	name = "power crepe"
+	desc = "Warning: May contain dairy products, 12,000kw of searing death, gluten."
+	origin_tech = Tc_POWERSTORAGE + "=3"
+	icon_state = "power_crepe"
+	maxcharge = 12000
+	charge = 12000
+	w_type = RECYK_BIOLOGICAL
+	minor_fault = 1
+
+/obj/item/weapon/cell/crepe/attack_self(var/mob/living/user)
+	user.visible_message("<span class = 'notice'>\The [user] takes a bite out of \the [src]</span>", "<span class = 'warning'>You take a bite out of \the [src]</span>")
+	spawn(rand(1,3) SECONDS)
+		var/power_to_use = min(charge, rand(800,1200))
+		playsound(loc, 'sound/effects/eleczap.ogg', 80, 1)
+		use(power_to_use)
```

Comment:
The sound effect and the rest of the code should be indented under an `if` for this - if the cell can't give you the power you want, this returns a 0.